package.json

```diff
@@ -64,18 +64,17 @@
   },
   "dependencies": {
     "ethereumjs-util": "^6.1.0",
-    "keythereum": "^1.0.4",
     "lodash.get": "^4.4.2",
     "nahmii-contract-abstractions": "2.0.1",
-    "nahmii-contract-abstractions-ropsten": "^3.1.0",
-    "nahmii-ethereum-address": "^2.0.0",
-    "node-yaml": "^3.2.0",
+    "nahmii-contract-abstractions-ropsten": "~3.1.0",
```

Comment:
Is going from to `^` to `~` intentional?

Reply:
hmm no.. good catch